### DISCUSSION TITLE
`dex.trades` redesign open discussion

### DISCUSSION THREAD

**Discussion by jeff-dude** [duneanalytics/spellbook#3917] [Sector: DEX]
todo: fill in full details, when unblocked by the migration to dunesql on spellbook
similar approach as #3317

**Comment by denverbaumgartner** [comment]
it would be great to have the following fields within `dex.trades`: 1) `block_number`, 2) `block_index` or `txn_index` (semantics but the same thing, 3) `evt_index`. this would help when trying to do anything related to ordering, as multiple trades can happen not just in the same block, but the same transaction as well.

**Reply by jeff-dude** [reply]
noted

**Comment by jeff-dude** [comment]
we're actively working on beta approach to `dex.trades` redesign in #4533 -- note that it's WIP and we plan to provide a `readme` within to talk through the details involved

**Comment by jeff-dude** [comment]
closing this in favor of #4637 as we look to clean up issues page and use that for new spells / designs / etc
